fastlane_core/lib/fastlane_core/print_table.rb

```diff
@@ -34,16 +41,60 @@ def print_values(config: nil, title: nil, hide_keys: [], mask_keys: [])
         return params
       end
 
-      def limit_row_size(rows, max_length = 100)
+      def colorize_array(array, colors)
+        value = ""
+        array.each do  |l|
+          colored_line = l
+          colored_line = "#{colors.first[0]}#{l}#{colors.last[0]}" if colors.length > 0
+          value << colored_line
+          value << "\n"
+        end
+        return value
+      end
+
+      def transform_output(rows, transform)
         require 'fastlane_core/string_filters'
+        require 'io/console'
+
+        return_array = []
+        tcols = IO.console.winsize[1]
+
+        col_count = rows.map(&:length).first || 1
+
+        # -4 per column - as tt adds "| " and " |"
+        terminal_table_padding = 4
+        max_length = tcols - (col_count * terminal_table_padding)
+
+        max_value_length = (max_length / col_count)
```

Comment:
I tend to only prefer them when there are multiple expressions to make precedence easier to read, but sure.